apps/api/src/index.ts

```diff
@@ -50,15 +50,16 @@ async function main() {
 
   const initOptions = {
     connectionString: dbUrl,
-    ssl:
-      config().NODE_ENV !== 'development' && !config().POSTGRES_SSL_DISABLED
```

Comment:
I don't think this works, when running locally I get this now:
```
@briefer/api:dev: [12:24:46.483] ERROR (42023): Uncaught error while starting server
@briefer/api:dev:     err: {
@briefer/api:dev:       "type": "Error",
@briefer/api:dev:       "message": "The server does not support SSL connections",
@briefer/api:dev:       "stack":
@briefer/api:dev:           Error: The server does not support SSL connections
@briefer/api:dev:               at Socket.<anonymous> (/Users/lucas/code/ergomake/briefer/packages/database/node_modules/pg/lib/connection.js:76:37)
@briefer/api:dev:               at Object.onceWrapper (node:events:632:26)
@briefer/api:dev:               at Socket.emit (node:events:517:28)
@briefer/api:dev:               at Socket.emit (node:domain:489:12)
@briefer/api:dev:               at addChunk (node:internal/streams/readable:335:12)
@briefer/api:dev:               at readableAddChunk (node:internal/streams/readable:308:9)
@briefer/api:dev:               at Socket.Readable.push (node:internal/streams/readable:245:10)
@briefer/api:dev:               at TCP.onStreamRead (node:internal/stream_base_commons:190:23)
@briefer/api:dev:     }
```
Am I missing something? Are you suggesting that I set `POSTGRES_SSL_DISABLED` to `true` when running locally? Why not keep the `config().NODE_ENV !== 'development'`?